gensrc/thrift/PlanNodes.thrift

```diff
@@ -214,6 +214,9 @@ struct THdfsScanRange {
 
     // file format of hdfs file
     6: optional Descriptors.THdfsFileFormat file_format
+
+    // property like 'field.delim' and 'line.delim'
+    7: optional map<string, string> properties
```

Comment:
How about create a TextFileFormatDesc for TextFile.
And store it in THdfsScanRange

field_delim and line_delim can be stored in THdfsScanRange